src/banman.h

```diff
@@ -37,6 +37,23 @@ class CSubNet;
 // dangerous, because it can cause a network split
 // between nodes running old code and nodes running
 // new code.
+//
+// With regards to impersonation attack, i.e a malicious
+// actor sending us directly invalid data while IP-spoofing
+// an address of honest reacheable peers, thus triggering an
+// unlawful ban of those:
```

Comment:
"unlawful"?

Reply:
Yes I need a better wording here, with regards to banning a misbehavior is qualified but in reality both peers are honest ?

Reply:
undesirable?